README.md

```diff
@@ -30,9 +30,17 @@ in another terminal if you are doing front end work:
 
     gulp watch
 
-
+Also note, that the staticfiles folder is the destination of all static assets when you or a script runs `manage.py collectstatic` so don't make your changes there, or they will be overwritten.
 
 ## cloud.gov set up
+You only need to get the services stood up, configure the S3 bucket once.
```

Comment:
This might be clearer as: "You only need to get the services stood up and configure the S3 bucket once."